auth/auth.go

```diff
@@ -177,13 +177,24 @@ func (c *Client) CustomTokenWithClaims(uid string, devClaims map[string]interfac
 	return encodeToken(c.snr, defaultHeader(), payload)
 }
 
+// RevokeRefreshTokens revokes all refresh tokens for the specified user.
+//
+// RevokeRefreshTokens revokes all refresh tokens for the specified user.
+// In addition to revoking all refresh tokens for a user, all ID tokens issued
+// before revocation will also be revoked at the Auth backend. Any request with an
```

Comment:
Hmmm... I don't think this is correct. Can we align this with what's documented at https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#revokeRefreshTokens ?